libs/dh/wholesale/feature-start/src/lib/dh-wholesale-start.component.html

```diff
@@ -16,8 +16,32 @@
 -->
 <ng-container *dhFeatureFlag="'start_wholesale_process_feature_flag'">
   <ng-container *transloco="let transloco; read: 'wholesale'">
-    <p>{{ transloco("startProcess.processDescription") }}</p>
-    <watt-button (click)="createBatch()"
+    <div class="date-range-width">
+      <watt-form-field>
+        <watt-label>{{
+          transloco("startProcess.gridAreaDropDownLabel")
+        }}</watt-label>
+        <watt-dropdown
+          [formControl]="formControlGridArea"
+          [options]="optionsGridAreas"
+          [showResetOption]="false"
+          [multiple]="true"
+        ></watt-dropdown>
+      </watt-form-field>
+      <watt-form-field>
+        <watt-label>{{ transloco("startProcess.dateRangeLabel") }}</watt-label>
+        <watt-datepicker
+          [formControl]="formControlRange"
+          [range]="true"
+        ></watt-datepicker>
+        <watt-error *ngIf="formControlRange?.errors?.rangeRequired">
+          {{ transloco("startProcess.dateRangeRequired") }}
+        </watt-error>
+      </watt-form-field>
+    </div>
+    <watt-button
+      (click)="createBatch()"
+      [disabled]="formControlRange.invalid || formControlGridArea.invalid"
       >{{ transloco("startProcess.startLabel") }}
     </watt-button>
```

Comment:
question: the user can still click the button if a request is going. This will cancel the current request and start a new one. Should the button be disabled? Another thing we can do is let the current request complete and not start a new one even if button is clicked several times. This will be quite easy actually. Just replace `switchMap` with `exhaustMap` in the store. 

question: should something happen after the request is done? Like a notification or

Reply:
At this time it is ok that the start button can be pushed multiple times, at a later date we will address this. The same goes for notification when the request is done

Reply:
Although the one about switchMap, I will change it to exhaustMap, because it makes sense that the first click has priority